packages/client/src/generation/utils/types/dmmfToTypes.ts

```diff
@@ -0,0 +1,18 @@
+import type { DMMF } from '../../../runtime/dmmf-types'
+import { TSClient } from '../../TSClient/TSClient'
+
+// Internal, used by the PDP not to need a call to the CLI
```

Comment:
https://tsdoc.org/pages/tags/internal/

```suggestion
/**
  * @internal
  *
  * @remarks Used by, for example, the PDP to avoid child process calls to the CLI.
  */
```